aten/src/ATen/native/native_functions.yaml

```diff
@@ -2916,6 +2921,9 @@
   dispatch:
     DefaultBackend: permute
 
+- func: cov(Tensor self, Tensor? other=None, Tensor? fweights=None, Tensor? aweights=None, bool? bias=None, int? ddof=None) -> Tensor
+  use_c10_dispatcher: hacky_wrapper_for_legacy_signatures
```

Comment:
I think this can ditch the hacky wrapper.

Reply:
I tried doing it without using the hacky wrapper but then it throws error reqd. difference in schema.
That is mostly due to the fact that I have mentioned everything other then self as optional in native_functions but have not mentioned the tensors as optional<Tensor> in the function definition.

I followed `torch.stft` as that function also used optional tensors in this fashion using the hacky wrapper.
What could I do in such a situation?